scripts/prepare.js

```diff
@@ -27,6 +27,10 @@ downloadIfNeeded(
   "http://repo1.maven.org/maven2/org/sonarsource/python/sonar-python-plugin/1.8.0.1496/sonar-python-plugin-1.8.0.1496.jar",
   "analyzers/sonarpython.jar"
 );
+downloadIfNeeded(
+  "https://repox.sonarsource.com/sonarsource/org/sonarsource/typescript/sonar-typescript-plugin/1.5.0.2021/sonar-typescript-plugin-1.5.0.2021.jar",
```

Comment:
@henryju we will update these build numbers (ts and sonarlint-ls) before release, right?